admin/kernel-syncdata-service/src/main/java/io/mosip/kernel/syncdata/controller/WebsubCallbackController.java

```diff
@@ -38,7 +35,7 @@ public class WebsubCallbackController {
     private static final String CERTIFICATE_DATA_SHARE_URL = "certChainDatashareUrl";
     private static final String PARTNER_DOMAIN = "partnerDomain";
 
-    @Autowired
+    @Mock
```

Comment:
Don't mock any object in code other than test classes.

Reply:
changed into Autowired